tutorials/skcsd_tutorial.ipynb

```diff
@@ -33,9 +44,32 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
+   "execution_count": 3,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Load /home/chchinta/anaconda3/envs/testinstall/lib/python3.8/site-packages/kcsd/data/gang_7x7_200/morphology/gang_7x7_200_rows_4_cols_5_xmin_-20_xmax_20_ymin_-20_ymax_20_orientation_3.swc\n"
+     ]
+    },
+    {
+     "ename": "TypeError",
```

Comment:
Could you upload version which executes correctly? The code works for me without errors

Reply:
I will update the correct version in the subsequent versions, i need to get the pypi installation working first and I will be able to resolve bunch of these issues in one go.

Reply:
Ok, so I will merge it as it is for now